azure-pipelines.yml

```
@@ -18,6 +18,8 @@ variables:
   # To make sure we don`t use MsBuild 17.
   - name: MsBuildPath
     value: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\msbuild.exe'
+  - name: isMaster
```

Early comment on WIP PR: `master` OR `branch-*`

So maybe something like
```suggestion
  - name: isReleaseBranch
```